iidm/iidm-xml-converter/src/test/resources/V1_1/VscRoundTripRef.xml

```diff
@@ -5,7 +5,7 @@
             <iidm:busBreakerTopology>
                 <iidm:bus id="B1"/>
             </iidm:busBreakerTopology>
-            <iidm:vscConverterStation id="C1" name="Converter1" lossFactor="0.011" voltageRegulatorOn="true" voltageSetpoint="405.0" bus="B1" connectableBus="B1" p="100.0" q="50.0">
+            <iidm:vscConverterStation id="C1" name="Converter1" voltageRegulatorOn="true" lossFactor="0.011" voltageSetpoint="405.0" bus="B1" connectableBus="B1" p="100.0" q="50.0">
```

Comment:
It's strange that the order of the two attributes is changing

Reply:
Revert.